<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="VictorEspinoza" name="com.regression.Test_2" tests="1" failures="0" timestamp="2020-05-04T17:06:30 MDT" time="3.188" errors="1">
  <testcase name="f" time="3.188" classname="com.regression.Test_2">
    <error type="java.lang.Error" message="Unresolved compilation problems: 
	data cannot be resolved
	data cannot be resolved
	data cannot be resolved
	data cannot be resolved
	data cannot be resolved
	data cannot be resolved
	data cannot be resolved
">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
data cannot be resolved
data cannot be resolved
data cannot be resolved
data cannot be resolved
data cannot be resolved
data cannot be resolved
data cannot be resolved

at com.regression.Test_2.f(Test_2.java:25)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- f -->
  <system-out/>
</testsuite> <!-- com.regression.Test_2 -->
